src/calculateRentalCost.js

```diff
@@ -28,7 +28,19 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  if (days === 7) {
+    return 230;
+  }
+
+  if (days === 6) {
+    return 220;
+  }
+
+  if (days === 3) {
+    return 100
+  }
+
+  return 80;
 }
```

Comment:
pay attention on [checklist](https://github.com/mate-academy/js_task-transportation-on-vacation/blob/master/checklist.md)
all values from task description write inte constants and use them for return some value.